spec/rabbitmqHandlersSpec.js

```diff
@@ -0,0 +1,129 @@
+const rabbitMqHandlers = require('../src/handlers/rabbitmq-handlers')
+// Test valid rabbitmq-handlers input
+describe('rabbitmq-handlers', () => {
+  // it('should have valid content', (done) => {
```

Comment:
If the commented code is not going to be used, it should be deleted

Reply:
Fixed